tools/reset_github_user_token_and_update_circleci.sh

```diff
@@ -45,14 +60,6 @@ then
     USERNAME="${USERNAME:-$DEFAULT_USERNAME}"
 fi
 
-if [[ -z "$PASSWORD" ]];
-then
-    echo "What is the password for the GitHub user (${USERNAME})?"
-    echo "This password is stored in LastPass under the ${DEFAULT_USERNAME}"
-    echo "account."
-    read -s PASSWORD
-fi
-
```

Comment:
Wait a minute, at which point Github Password is not used anymore? What's used instead then and where it's documented?

Reply:
As of #106, `$PASSWORD` is no longer used in the script, so there is no need to prompt for it.

#106 was necessary because [Github disabled basic auth with user/password](https://developer.github.com/changes/2020-02-14-deprecating-password-auth/), so we cannot use it to automatically generate the PATs any more. [This line](https://github.com/StackStorm-Exchange/ci/pull/106/files#diff-5b060d9c9235f7c266766f99962be0fa99e168790c367228eb4ba3e30e2fe0b5L59) is where we used to use `$PASSWORD`.

Reply:
That makes sense.
Can we include that info really briefly somewhere in the script header?